### PR TITLE
Prevents blob from growing over asteroid walls

### LINKED ISSUE #584: Green blobs actually don't destroy rock walls but grow in/through them.
https://gyazo.com/d2db4343e044a00984b10043c6d0922b

### PR DESCRIPTION
**Pull request by Hubblenaut**
Fixes #584

### CODE REVIEW

File: code/modules/blob/blob.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@
 	update_icon()
 
 /obj/effect/blob/proc/expand(var/turf/T)
-	if(istype(T, /turf/unsimulated/) || istype(T, /turf/space))
+	if(istype(T, /turf/unsimulated/) || istype(T, /turf/space) || istype(T, /turf/unsimulated/mineral))
```
Comment (Techhead0) [inline code/modules/blob/blob.dm]: Shouldn't `istype(T, /turf/unsimulated/)` return true for `/turf/unsimulated/mineral` anyway?

Comment (Zuhayr) [inline code/modules/blob/blob.dm]: Yes.

Comment (Hubblenaut) [inline code/modules/blob/blob.dm]: My mistake, this is supposed to be simulated, not unsimulated.

**Comment by Zuhayr** [comment]
You will need to cast to /turf/simulated/mineral and check density, a change was merged that uses the same turf for all kinds of asteroid, with density determining if it's a wall or not.

**Pull request merged**